protocol/0062-SPAM-spam_protection.md

```diff
@@ -89,6 +89,9 @@ Network parameter: `spam.order.MinimalMarginQuantumMultiple` (between 0 and infi
 
 If the maintenance margin for a given transaction is smaller than the parameter `spam.order.MinimalMarginSizeQuantumMultiple`, then the transaction is pre-block rejected.
 I.e. if `(rf + linear slippage param) x size x price <  spam.order.MinimalMarginSizeQuantumMultiple x asset quantum amount` then the order is rejected. Here `rf` is the risk factor (and will be different for long and for short) `linear slippage param` is a market parameter and `size` and `price` are assumed to be correctly scaled by, PDPs and MDPs respectively.
+- For peg orders we check the `price` implied by the peg and the book at submission time. 
```

Comment:
what if the peg doesn't exist?

Reply:
and during an auction.

Reply:
Resolving as PR updated to use last traded price as an estimate instead of exact peg.